### PR TITLE
[feat]submit cpu topology feature firstly

### PR DESCRIPTION
**Pull request by huone1**

### CODE REVIEW

File: config/crd/bases/scheduling.volcano.sh_podgroups.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.2.4
```
Comment (william-wang) [inline config/crd/bases/scheduling.volcano.sh_podgroups.yaml:7]: why change to use a lower version?

File: config/crd/bases/scheduling.volcano.sh_podgroups.yaml [hunk 2]
```diff
@@ -48,11 +48,7 @@ spec:
                 type: integer
               minResources:
                 additionalProperties:
-                  anyOf:
-                  - type: integer
-                  - type: string
-                  pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
-                  x-kubernetes-int-or-string: true
+                  type: string
```
Comment (william-wang) [inline config/crd/bases/scheduling.volcano.sh_podgroups.yaml:51]: what's the reason to change the pattern?

File: config/crd/bases/scheduling.volcano.sh_podgroups.yaml (hunk 1 above)
Comment (huone1) [reply config/crd/bases/scheduling.volcano.sh_podgroups.yaml:7]: in my mechine, run "make update-development-yaml", auto change it to v0.2.4, and use current yaml in master to install volcano， have a error in job api
image

File: config/crd/bases/scheduling.volcano.sh_podgroups.yaml (hunk 2 above)
Comment (huone1) [reply config/crd/bases/scheduling.volcano.sh_podgroups.yaml:51]: controller-gen version is different between github  and my env， I update my env to v0.4.1

File: config/crd/bases/scheduling.volcano.sh_podgroups.yaml (hunk 1 above)
Comment (huone1) [reply config/crd/bases/scheduling.volcano.sh_podgroups.yaml:7]: controller-gen version is different between github  and my env， I update my env to v0.4.1

**Comment by huone1** [comment]
close it, submit a new PR #1457

**Pull request closed without merging**